input/pagecontent/etlTalent.md

```diff
@@ -0,0 +1,216 @@
+This section describes how to build an ETL-process and fill the Blaze Store with Talend Open Studio and Hapi FHIR Api. This instructions were [originally](https://wiki.verbis.dkfz.de/pages/viewpage.action?pageId=76351392) writen and tested by Marko Wegehaupt from Wuerzburg in november 2019.
+
+## Step-by-step instructions:
+1. A new routine must be created in [Talend Open Studio](https://www.talend.com/products/talend-open-studio/). In this manual it is called FHIR_API_Wrapper. click on „Edit libraries of the Routines“ and import the following libraries from [hapi-fhir-4.0.0-standard-distribution](https://github.com/jamesagnew/hapi-fhir/releases) to solve imports.
+![TOS import external libs](../images/TOS_FHIR_01_libraries.png "Import external libs in TOS")
+
+2. Use JavaRow to create a FHIR_API_Wrapper Object in Talent and run methodes on it to produce FHIR Resources.
+3. In a final step upload the generated Bundles to the Blaze Store.
+
+## In details:
```

Comment:
No need to enumerate the Points in this next section. Because they are headings, the Publisher already prefixes them with 7.1 - 7.6